src/navigation/actions.js

```diff
@@ -266,26 +266,27 @@ export const gotoStocktakes = () =>
  * Pushes the Stock route onto the main navigation stack.
  */
 export const gotoStock = () =>
-  NavigationActions.navigate({
-    routeName: ROUTES.STOCK,
+  CommonActions.navigate({
+    name: ROUTES.STOCK,
     params: {
       title: navStrings.current_stock,
     },
   });
 
 /**
- * Action creator for navigating to the SupplierRequisition screen.
+ * Action creator for navigating to the Stocktake Manage screen.
  *
- * @param {Object} requisition The requisition to pass to the next screen.
+ * @param {String} stocktakeName The name of the stocktake to pass to the next screen.
+ * @param {Object} stocktake The stocktake to pass to the next screen.
  */
 export const gotoStocktakeManagePage = (stocktakeName, stocktake) => dispatch => {
-  const currentRouteName = RootNavigator.getCurrentRouteName();
+  const currentName = RootNavigator.getCurrentRouteName();
 
   const navigationActionCreator =
-    currentRouteName === ROUTES.STOCKTAKES ? NavigationActions.navigate : StackActions.replace;
+    currentName === ROUTES.STOCKTAKES ? CommonActions.navigate : StackActions.replace;
```

Comment:
This ones throwing an error for me and won't let me do a normal stocktake.

Looks like the `StackActions` api has changed also https://reactnavigation.org/docs/stack-actions 

Seems they `CommonActions` able to pass `name, params` or `{ name, params }` but for `StackActions` it is only `name, params` ?

Reply:
Thanks! That was subtle. Have fixed with 3 commits (24c80c8) was the main one but then I did some copy paste fails so there's also cb3b72f and bfbfe79.

Found some other strange behaviour where deleting multiple unfinalised stocktakes causes a rsod, but I'm not sure if it was this PR yet, will give it a test on develop (and master if it also happens there).